core/block_svg.js

```diff
@@ -986,10 +986,9 @@ Blockly.BlockSvg.prototype.setBorderColour_ = function(rgb, colourTertiary) {
  * Sets the colour of shadow blocks.
  * @param {!string} rgb Primary colour of the block.
  * @param {?string} colourSecondary Colour for shadow block.
- * @return {!string} hexColour The background color of the block.
+ * @return {!string} hexColour The background colour of the block.
```

Comment:
Remove `hexColour`. Return values don't have names.

Reply:
Fixed this and 65 other @return annotations.